crates/hyperdrive-math/src/long/targeted.rs

```diff
@@ -348,7 +360,6 @@ mod tests {
     use super::*;
     use crate::test_utils::agent::HyperdriveMathAgent;
 
-    #[ignore]
```

Comment:
I ran the test locally 3 times and it passed every time.

Reply:
![image](https://github.com/delvtech/hyperdrive-rs/assets/5555162/95dec73e-22ed-4341-807b-81706c1a045b)